src/hooks/index.ts

```diff
@@ -728,6 +726,11 @@ export const useIsOnline = (
     refetchInterval: 5000,
     suspense: true,
     useErrorBoundary: false,
+    onSuccess: (isOnline) => {
+      if (wasOnline.current === isOnline) return
```

Comment:
bugfix
only set online if status changes
this was causing queries to refetch over and over